cmd/ddev/cmd/config.go

```diff
@@ -107,7 +107,10 @@ var ConfigCommand = &cobra.Command{
 				if _, err = os.Stat(docrootRelPath); os.IsNotExist(err) {
 					util.Failed("The docroot provided (%v) does not exist", docrootRelPath)
 				}
+			} else if !cmd.Flags().Changed("docroot") {
+				app.Docroot = ddevapp.DiscoverDefaultDocroot(app)
```

Comment:
@rfay This is my workaround. If the `docroot` was not changed, default to detected directory.

Reply:
Looks correct to me.